File: BoSeonPark/그리디 알고리즘/11399.py
import sys
input = sys.stdin.readline

n = int(input())
timeTalbe = list(map(int, input().split(' ')))

timeTalbe.sort()
total = 0
a = 0
for t in timeTalbe:
    a += t
    total += a

print(total)
    

File: 김하영/Barkingdog_Greedy/11399.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "input.txt";

const input = fs.readFileSync(filePath).toString().trim().split("\n");
const N = Number(input[0]);
const times = input[1]
  .split(" ")
  .map(Number)
  .sort((a, b) => a - b);

const solution = () => {
  let result = 0;
  const arr = [];
  times.forEach((time) => {
    result += time;
    arr.push(result);
  });

  return arr.reduce((prev, curr) => {
    return prev + curr;
  }, 0);
};

console.log(solution());
